specs/pvs/corla.pvs

```diff
@@ -248,14 +248,22 @@ elections: THEORY
 BEGIN
 %%% Voters are the most important concept in elections, as they are
 %%% who elections are for, and who make the choices that determine the
-%%% outcome of legitimate elections. Some voters are UOCAVA
-%%% voters. What follows is one way to formalize such, simply stating
-%%% that the type uocava\_voter is a predicate on the type
-%%% voter. Consequently, all voters are either UOCAVA or not.
-  person: TYPE
-  voter: TYPE FROM person
+%%% outcome of legitimate elections.
+%%%
+%%% Voters have first and lastnames and a political party affiliation.
+  person: TYPE =
+    [# firstname: string, lastname: string #]
+  political_party: TYPE = string
+  voter: TYPE = %FROM person
+    [# firstname: string,
+       lastname: string,
+       party_affiliation: political_party #]
   elector: TYPE = voter
-  uocava_voter: TYPE from voter
+%%% Some voters are UOCAVA voters. What follows is one way to
+%%% formalize such, simply stating that the type
+%%% \texttt{uocava\_voter} is a predicate on the type
+%%% voter. Consequently, all voters are either UOCAVA or not.
+  uocava_voter: pred[voter]
```

Comment:
I know this commit is merged, so ignore this comment if desired.  But I'm unclear on the value of the term uocava_voter, or the modeling of a firstname and lastname but nothing else. I don't see the term UOCAVA in the draft rule, and non-uocava ballots can come from uocava voters. So I'm unclear on the benefit of these sections. I think from the auditing perspective, non_voter_verifiable_ballot describes what we need to deal with, and even that maybe not this go round.